prow/plugins/skip/skip_test.go

```diff
@@ -183,7 +183,8 @@ func TestSkipStatus(t *testing.T) {
 					},
 				},
 			},
-			sha: "shalala",
+			sha:            "shalala",
+			combinedStatus: github.StatusSuccess,
```

Comment:
I think it would be better to have this as a separate case?

Reply:
Thank you for the review. I have updated the PR.